keystore/src/connection/platform/generic/migrations/V12__unique_id_constraints.sql

```diff
@@ -0,0 +1,13 @@
+---- mls_encryption_keypairs ----
+CREATE TABLE mls_encryption_keypairs_new (
+    id_hex TEXT UNIQUE,
+    pk BLOB,
+    sk BLOB
+);
+
+INSERT INTO mls_encryption_keypairs_new (id_hex, pk, sk)
+SELECT lower(hex(pk)), pk, sk FROM mls_encryption_keypairs;
+
+DROP TABLE mls_encryption_keypairs;
+
+ALTER TABLE mls_encryption_keypairs_new RENAME TO mls_encryption_keypairs;
```

Comment:
What was the motivation for this? Maybe to allow for fast lookups based on the hex ID?